src/components/Map/index.js

```diff
@@ -24,102 +27,119 @@ const Skater = new Icon({
+                                            </span>{value.swimmingpool}</p>
                                         </div>
                                     </Grid>                
                                     <Grid item xs>
                                         <div>
-                                        <p className={classes.spanText} style={{fontSize: '0.5rem', marginTop: -5}}><span className={classes.spanIcon}>{hotelCoordinate.parking.icon}</span>{hotelCoordinate.parking.label}</p>
+                                        <p className={classes.spanText} style={{fontSize: '0.5rem', marginTop: -5}}><span className={classes.spanIcon}>
+                                        <LocalParkingIcon style={{width:20,height: 13,}}/>
+                                        </span>{value.parking.label}</p>
                                         </div>
                                     </Grid>                
                             </Grid>
                             <Grid item sm={12} container>
                                     <Grid item xs>
                                         <div>
-                                            <p className={classes.spanText} style={{fontSize: '0.5rem', marginTop: -20}}><span className={classes.spanIcon}>{hotelCoordinate.wifi.icon}</span>{hotelCoordinate.wifi.label}</p>
+                                            <p className={classes.spanText} 
+                                                style={{fontSize: '0.5rem', marginTop: -20}}
+                                            >
+                                                <span className={classes.spanIcon}>
+                                                    <WifiIcon style={{width:20,height: 13,}}/>
+                                                    
+                                                </span>
+                                                    {value.wifi}
+                                            </p>
                                         </div>
                                     </Grid>                
                                     <Grid item xs>
                                         <div>
-                                        <p className={classes.spanText} style={{fontSize: '0.5rem', marginTop: -20}}><span className={classes.spanIcon}>{hotelCoordinate.breakFast.icon}</span>{hotelCoordinate.breakFast.label}</p>
+                                        <p className={classes.spanText} style={{fontSize: '0.5rem', marginTop: -20}}><span className={classes.spanIcon}>
+                                        <FreeBreakfastIcon style={{width:20,height: 13,}}/>
+                                        </span>{value.breakfast}</p>
                                         </div>
                                     </Grid>
                             </Grid>
                             <Link to="hotel/more" style={{textDecoration: 'none'}} > 
```

Comment:
Could you please  link this availability to booking page since a user will be checking the availability of rooms so s/he can book